go.mod

```diff
@@ -5,7 +5,7 @@ go 1.17
 require (
 	github.com/alecthomas/units v0.0.0-20210927113745-59d0afb8317a
 	github.com/blang/semver/v4 v4.0.0
-	github.com/brancz/kube-rbac-proxy v0.8.0
+	github.com/brancz/kube-rbac-proxy v0.11.0
```

Comment:
Do we use krb in the project?

Reply:
Yes - https://github.com/prometheus-operator/prometheus-operator/blob/e9861e7bdb2990062d59ccfb59cc016b1cf30dd2/cmd/operator/main.go#L43